plugins/admin/README.adoc

```diff
@@ -84,7 +84,8 @@ Available Commands:
   remove      Remove registry settings
 
 Flags:
-  -h, --help   help for registry
+  -h, --help               help for registry
```

Comment:
Inconsistent capitalization "help" and "Namespace". Start with capital like with `kn`

Reply:
@maximilien Thank you for pointing this out and I totally agree with you. I just generally updated all the outputs of registry cmd to correct the formats and sync with `kn` styles. Also as @zhanggbj suggested I added `serviceaccount` flag in this pr and the issue also. Please feel free to review again and leave any comments!